File: Iniciante/3040/3040.c
#include <stdio.h>

int main()
{

    int casos, altura, tronco, galhos, i;

    scanf("%d", &casos);

    for (i = 0 ; i < casos; ++i)
    {

        scanf("%d %d %d", &altura, &tronco, &galhos);

        if (altura >= 200 && altura <= 300)
            if (tronco >= 50)
                if (galhos >= 150)
                {
                    printf("Sim\n");
                    continue;
                }

        printf("Nao\n");

    }

    return 0;

}

File: Python/Iniciante/3040.py
# Problema 3040 - URI - Iniciante - Nível 1

# Recebendo o número de casos de testes do problema.
numTestes = int(input())

# Iniciando um vetor, que armazenará as respostas para os testes
respostas = []
for i in range(numTestes):
# Para cada teste, recebendo o valor da altura da árvore, o diámetro e a quantidade de galhos.
  alt, dmt, qtdGalhos = map(int, input().split(" "))

# Decidindo se os parámetros da árvore são adequados de acordo com o problema, e guardando a resposta em um vetor de respostas.
  if(alt>=200 and alt<=300 and dmt>=50 and qtdGalhos>=150):
    respostas.append("Sim")
  else:
    respostas.append("Nao")

# Apresentando as respostas
for i in range(numTestes):
  print(respostas[i])